test/test.store.js

```diff
@@ -36,3 +35,39 @@ tap.test('store plugin calls redis server', (t) => {
     });
   });
 });
+
+tap.test('store plugin can scan for matching keys', (t) => {
+  async.autoInject({
+    rapptor(done) {
+      const rapptor = new Rapptor({ env: 'test' });
+      rapptor.start(done);
+    },
+    setup(rapptor, done) {
+      server = rapptor[0];
+      return done(null, server);
+    },
+    scan1(setup, done) {
+      t.equal(typeof server.store.scan, 'function', 'store.scan is registered');
+      server.methods.getKeys('key', done);
+    },
+    verify1(scan1, done) {
+      t.equal(scan1.length, 1);
+      done();
+    },
+    scan2(setup, done) {
+      server.store.set('key1', 'value1');
+      server.store.set('key2', 'value2');
+      server.store.set('key3', 'value3');
```

Comment:
You need to add something in here that doesn't match

Reply:
Also, is set a syncronous call?

Reply:
What about this?

Reply:
Yeah set is synchronous in the ioredis docs, I think we can trick it into being async using ioredis's pipeline if needed.

Reply:
https://github.com/luin/ioredis/issues/279 all cleared up